stable/mssql-linux/templates/pvc-tranlog.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingTransactionLogClaim) }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "mssql.fullname" . }}-log
+  labels:
+    app: {{ template "mssql.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+{{- if .Values.persistence.annotations }}
+  annotations:
+{{ toYaml .Values.persistence.annotations | indent 4 }}
+{{- end }}
+spec:
+  accessModes:
+    - {{ .Values.persistence.transactionLogAccessMode | quote }}
+  resources:
+    requests:
+      storage: {{ .Values.persistence.transactionlogSize | quote }}
```

Comment:
I think theres a Typo here - the L in Log should be uppercase to match the values.yaml setting

Reply:
Thanks for pointing this out.  Corrected